picoquictest/stresstest.c

```diff
@@ -966,7 +966,7 @@ static int stress_or_fuzz_test(picoquic_fuzz_fn fuzz_fn, void * fuzz_ctx, uint64
     /* Initialization */
     memset(&stress_ctx, 0, sizeof(picoquic_stress_ctx_t));
     stress_set_ip_address_from_index(&stress_ctx.server_addr, -1);
-    stress_ctx.nb_clients = (int)picoquic_stress_nb_clients;
+    stress_ctx.nb_clients = nb_clients;
     if (stress_ctx.nb_clients > PICOQUIC_MAX_STRESS_CLIENTS) {
         DBG_PRINTF("Number of stress clients too high (%d). Should be lower than %d\n",
```

Comment:
why not use nbclients here too, just like in the other lines of code further down?

Reply:
But then, it does not change the actual results...